phishing-pages/plugin-update/update/payloadFile.txt

```diff
@@ -0,0 +1 @@
+REPLACE THIS FILE WITH YOUR DESIRED PAYLOAD AND EDIT THE DOWNLOAD BUTTON ON THE INDEX PAGE TO POINT TO YOUR PAYLOAD
```

Comment:
No need to give directions here. If the payload file does not exist, it simply will return a 404.